modules/repository-azure/src/internalClusterTest/java/org/elasticsearch/repositories/azure/AzureStorageCleanupThirdPartyTests.java

```diff
@@ -89,8 +92,10 @@ protected SecureSettings credentials() {
         MockSecureSettings secureSettings = new MockSecureSettings();
         secureSettings.setString("azure.client.default.account", System.getProperty("test.azure.account"));
         if (hasSasToken) {
+            logger.info("Using SAS token authentication");
             secureSettings.setString("azure.client.default.sas_token", System.getProperty("test.azure.sas_token"));
         } else {
+            logger.info("Using key authentication");
```

Comment:
Nit: can we add `--> ` in the beginning of the logging messages? It's an informal conventional to make these test logging messages easier to search.

Reply:
Added in d9ce4b572db526458a3d99e8c82cfa3fee0d01a6